From 0eccde3eb907a74c46a58ff827bbfc2bb90dadce Mon Sep 17 00:00:00 2001
From: Victor Sun <victor.sun@intel.com>
Date: Wed, 5 Feb 2020 16:43:00 +0800
Subject: [PATCH] acrn-config: append kconfig setting on new board

When user has already prepared some default kconfig settings in
hypervisor/arch/x86/configs/$(BOARD).config file even the board is new
to Acrn source, the acrn-config will keep these settings and append other
suggested settings on the end of the $(BOARD).config file.

Signed-off-by: Victor Sun <victor.sun@intel.com>

Upstream-Status: Inappropriate 
Signed-off-by: Chee Yang Lee <chee.yang.lee@intel.com>
---
 misc/acrn-config/board_config/board_cfg_gen.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/acrn-config/board_config/board_cfg_gen.py b/misc/acrn-config/board_config/board_cfg_gen.py
index defd1bf..ffbef40 100755
--- a/misc/acrn-config/board_config/board_cfg_gen.py
+++ b/misc/acrn-config/board_config/board_cfg_gen.py
@@ -114,7 +114,7 @@ def main(args):

     # generate new board_name.config
     if need_gen_new_board_config(board):
-        with open(config_board_kconfig, 'w+') as config:
+        with open(config_board_kconfig, 'a+') as config:
             err_dic = new_board_kconfig.generate_file(config)
             if err_dic:
                 return err_dic
-- 
2.7.4
